lib/spack/spack/reporters/cdash.py

```diff
@@ -324,11 +357,7 @@ def test_report_for_package(self, report_dir, package, duration):
             return
 
         self.current_package_name = package["name"]
-        if self.base_buildname == self.install_command:
-            # The package list is NOT all that helpful in this case
-            self.buildname = "{0}-{1}".format(self.current_package_name, package["id"])
```

Comment:
Thanks for volunteering to review this PR for me, @tldahlgren !

I wanted to call your attention to this change in particular. Do you happen to recall the use case where this buildname format was needed?

Reply:
IIRC I was running tests on a package that had multiple versions (specs) installed and it was hard to map the output to the specific spec.  So this is more for command-line use.